usecases/recording.cameras.opencv.3/README.md

```diff
@@ -0,0 +1,72 @@
+This folder provides the instructions for testing proxy-camera, which can be executed from a docker-compose file to start all micro-services to record video streams from three OpenCV cameras with lossless H264 compression. It is assumed that git, Docker, and Docker Compose are installed and all the three cameras are properly connected. To install Docker, follow the tutorial: https://docs.docker.com/engine/installation/linux/ubuntulinux/
+
+### Prepare OpenDaVINCI Docker image
+
+Clone the OpenDaVINCI source tree and get the latest update:
+
+    $ git clone https://github.com/se-research/OpenDaVINCI
+    
+    $ git pull
+    
+Go to the docker folder and build OpenDaVINCI Docker image:
+
+    $ make
+    
+### Prepare proxy-camera
```

Comment:
@HangYin2015 The separate feature.camera branch will be deleted; please adjust the documentation here.